### PR TITLE
Added link to sonar home dashboard in sonar widget

### LINKED ISSUE #1122: Sonar analysis in Hygieia to Sonarqube Server
Hi there,
Is it possible to go to Sonarqube Server from the sonar Analysis in Hygieia Dashboard? like when we see the build of Jenkins job and click on the build then it redirect us to Jenkins Server.

Regards,
Praveen Kumar

### PR DESCRIPTION
**Pull request by brennanananan**
This PR adds a link to the Sonar instance for the project as determined by the collector. 

* The 'View All' link can be found on the widget near the header for the scan results.
* Link does not go to sonar page until data is loaded into the widget, since data has not been populated yet it would not know where to go.
* Tried to achieve same look and feel for links as in other places in application.

image

Resolves #1122

### CODE REVIEW

File: UI/src/components/widgets/codeanalysis/view.html [hunk 1]
```diff
@@ -2,7 +2,10 @@
     <div class="col-lg-4 col-sm-12">
         <div class="widget ca-issues">
             <div class="widget-heading">
-                Static Analysis
+            <span class="">Static Analysis</span>
+            	<a href="{{caWidget.reportUrl}}">
```
Comment (nireeshT) [inline UI/src/components/widgets/codeanalysis/view.html]: use target="_blank" so that "view all" will load in new page.

Comment (brennanananan) [inline UI/src/components/widgets/codeanalysis/view.html]: Fixed

**Comment by nireeshT** [comment]
@brennanananan  - As we enable the link in Sonar widget "View All", can we do a check on underlying url in DefaultSonarClient.java -> currentCodeQuality()

if(project.getInstanceUrl().endsWith("/")){
                    codeQuality.setUrl(project.getInstanceUrl() + "dashboard/index/" + project.getProjectId());
                }else{
                    codeQuality.setUrl(project.getInstanceUrl() + "/dashboard/index/" + project.getProjectId());
                }

Eg: 
http://localhost:9000/dashboard/index/1 ----> (valid url)
http://localhost:9000//dashboard/index/1 ----> (invalid url)

**Comment by brennanananan** [comment]
@nireeshT I have added the features as suggested. Thank you for the feedback.

**Comment by nireeshT** [comment]
@tabladrum  LGTM

**Comment by satishc1** [comment]
@brennanananan Thanks for the contribution.

**Comment by satishc1** [comment]
@brennanananan @tabladrum LGTM

**Pull request merged**